examples/vm-priorityclass.yaml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
these examples are auto generated by the tools/vms-generator/vms-generator.go  script.

I know that seems tedious. It's done that way because it guarantees that our examples are generated using the golang api structs, which keeps the examples updated automatically for us if anything changes.

Reply:
Thanks, already found this and working on this.